AnnoyancesFilter/sections/popups.txt

```diff
@@ -97,9 +97,6 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
-indirimdemarket.com#$##social-media-modal { display: none!important; }
-indirimdemarket.com#$#body { overflow: visible!important; }
-indirimdemarket.com#$#.modal-backdrop { display: none !important; }
```

Comment:
Don't remove it.

Reply:
Reverted.

Reply:
If the popup is social, we will add them both in Annoyances and Social filters.

Reply:
Okay.